src/webview/serverless-function/serverlessFunctionLoader.ts

```diff
@@ -47,7 +47,6 @@ async function gitImportMessageListener(panel: vscode.WebviewPanel, event: any):
             break;
         case 'createFunction':
             const selctedFolder: vscode.Uri = vscode.Uri.file(path.join(functionPath.fsPath, functionName));
-            panel.title = `Serverless Function - Create - ${functionName}`;
```

Comment:
@msivasubramaniaan There should be a title with `Serverless Function - Create`.

Reply:
yes it will get the title from https://github.com/redhat-developer/vscode-openshift-tools/blob/55431dd0575860b32b2f412ade2132341f8fd3e2/src/serverlessFunction/view.ts#L203C9-L203C9